lib/Models/Catalog/CatalogReferences/SplitItemReference.ts

```diff
@@ -51,7 +51,12 @@ export default class SplitItemReference extends ReferenceMixin(
     }
 
     try {
-      // Ensure the target we create is a concrete item
+      // Ensure the target we create is a concrete item and check if we can return previousTarget
+      if (
+        previousTarget !== undefined &&
+        sourceItem.uniqueId === this.splitSourceItemId
```

Comment:
What does this check guarantee? 

Also, it's probably not a standard use-case of SplitItemReference, but if `this.splitSourceItemId` changes between calls to `loadReference` this change will return the old model.

Reply:
This check is needed to prevent the second item (dragged from first to second in workbench) from stripping out the required dereferenced details, which is the cause of the bug.  i.e.  if there is a previousTarget passed in and the id's match return the previousTarget